clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/data/Keys.groovy

```diff
@@ -101,16 +107,32 @@ class Keys {
   }
 
   static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId, String loadBalancerType) {
+    return getLoadBalancerKey(loadBalancerName, account, region, vpcId, loadBalancerType, null)
+  }
+
+  static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId, String loadBalancerType, String loadBalancerArn) {
     String key = "${ID}:${Namespace.LOAD_BALANCERS}:${account}:${region}:${loadBalancerName}"
     if (vpcId) {
       key += ":${vpcId}"
       if (loadBalancerType && loadBalancerType != 'classic') {
         key += ":${loadBalancerType}"
+        if (loadBalancerArn != null) {
```

Comment:
There was something, but looks like I refactored whatever it was? Sure, that's it. I'll remove it.